mobile/src/controllers/LocalSettings.ts

```diff
@@ -26,6 +26,9 @@ export interface ILocalSettingsController {
     readonly synced: IEvent;
 
     updateNotifications(diff: Partial<NotificationsSettings>): void;
+    updateQolOnboarding(diff: Partial<QolSettings>): void;
```

Comment:
Same thing here about onboarding qol... there is no such thing. Why is this not just the same as other full qols?

Reply:
There is an explainer/welcome screen (I believe that is where `QolOnboarding` comes from) that is shown on the first survey right before the first question. I can rename this to something like initialQol

Reply:
Ok that naming actually makes sense because it's ending with the word "onboarding". Thanks!